### DISCUSSION TITLE
model training not using potential of graphics card

### DISCUSSION THREAD

**Discussion by pambroise** [coqui-ai/TTS#1467] [General Q&A]
Hello, I am trying to train a model, it's training but when I check gpu usage, I get this :
![Capture](https://user-images.githubusercontent.com/46627005/161370657-f2a65633-c538-46b9-86d0-79ebecc8c65f.PNG)

The model I am training is tacotron DCA 2 with default config and 500 files from LJSpeech dataset,  when using default batch size cuda tells me I am running out of memory as it is only using the base 8GB and not the 16GB of shared memory. To train my model I have to get the batch size down to like 16

On the screenshot you can also see GPU usage never goes above 15% and is on average below 5%

How do I use more memory/power to make my training more efficient and faster ?

**Comment by erogol** [comment]
I guess you are on windows. 

Do you use WSL or just native Windows to start training? 

(We don't support window natively unfortunately)

**Comment by pambroise** [comment]
I use native windows and had to make some tweaks to make the training work. I'll try to re-clone the repo and work on it from WSL then

**Reply by KyuubiYoru** [reply]
What changes did you make to get tacotron 2 to run on windows? I only got the other models to work.

**Comment by harmlessman** [comment]
Does the following phrase appear when you proceed with the learning?

 **> Using CUDA: True**
 **> Number of GPUs: 1**

If the number of gpus is 0, it could be a cuda problem.
Please re-install cuda&cudnn version that fits the gpu

**Comment by KyuubiYoru** [comment]
You need to switch one of the graphs to cuda to see the GPU load.
![image](https://user-images.githubusercontent.com/33806787/182765413-17d0ce9a-d51a-418e-9668-328c02408786.png)
